### ISSUE TITLE
Admission controller crash loop

### ISSUE THREAD

**Issue by agentreno** [anchore/kubernetes-admission-controller#8]
Hello, after installing the admission controller chart, I see a crash loop in the admission controller pod with the following logs:

```
# kubectl logs admission-anchore-admission-controller-864d9bddc7-gwc42
I0203 16:02:11.895831       1 main.go:550] Initializing configuration
I0203 16:02:11.895895       1 main.go:562] Loading configuration from /config/config.json
I0203 16:02:11.896328       1 main.go:580] Loading creds from /credentials/credentials.json                                                                                                                                                                                     I0203 16:02:11.897239       1 main.go:616] Starting server
I0203 16:02:12.277966       1 round_trippers.go:405] GET https://10.100.0.1:443/api/v1/namespaces/kube-system/configmaps/extension-apiserver-authentication 200 OK in 21 milliseconds
Error: cluster doesn't provide client-ca-file

<shortened output, looks like a CLI usage guide>

F0203 16:02:12.279599       1 cmd.go:42] cluster doesn't provide client-ca-file
```

I'm running this on AWS EKS with a git clone of this repo as the helm chart source.

**Comment by cplee** [comment]
@zhill will you be publishing a new release with this fix?

**Comment by zhill** [comment]
@cplee, yes, will be updating the image to v0.2.1 and updating the helm chart to match

**Comment by zhill** [comment]
the image has been updated, with tag v0.2.1, and the chart has been updated and published (v0.2.2), so a repo update in helm should allow you to install the fix.
